Yoast/wordpress-seo
# Fixes plugin icons on core updates page and design of changelog display

The banners are being designed, but since they're loaded via shortlinks, we can just change those shortlinks to change the banners.

## Context

Trying to make sure our plugins look better on the updates page. Also trying to reflect the compatibility data from Yoast SEO.

Before:

![Screenshot 2020-08-21 at 09 30 11](https://user-images.githubusercontent.com/487629/90864542-247a9d80-e391-11ea-9f6e-8de9c925a7aa.png)
<img width="1279" alt="Screenshot 2020-08-21 at 09 30 33" src="https://user-images.githubusercontent.com/487629/90864607-44aa5c80-e391-11ea-9b67-ace8932b8873.png">

After:
![Screenshot 2020-08-21 at 09 23 54](https://user-images.githubusercontent.com/487629/90864567-32c8b980-e391-11ea-8f5a-bd0ccbec7261.png)
![Screenshot 2020-08-21 at 09 21 40](https://user-images.githubusercontent.com/487629/90864569-33f9e680-e391-11ea-87de-50a85c750978.png)

## Summary

This PR can be summarized in the following changelog entry:

* Makes sure Yoast SEO premium and the add-ons get the proper plugin icon when shown in the plugin updates screen.
* Uses Yoast SEO free's compatibility data (when available) to display compatibility data for the add ons.

## Relevant technical choices:

* Used short URLs for the icons and banners so we can easily replace them.
* Had to replace `h3` and `h2` headings with `h4` headings in the changelog to make the design look good on the changelog screen.

## Test instructions

This PR can be tested by following these steps:

* Open the core updates page, make sure there are updates available for one or multiple add ons. See the updates page looks like shown above in "Before".
* Switch to this branch, see the updates page now looks like "After".
* Make sure updates still work properly.
* When Yoast SEO free is installed, compatibility data will show, if it's not installed, this will not show.

## UI changes

## Documentation

## Quality assurance


Fixes #
